malcolm/core/process.py

```diff
@@ -62,7 +63,7 @@ def recv_loop(self):
                 # Got the sentinel, stop immediately
                 break
             try:
-                self._handle_functions[request.type_](request)
```

Comment:
If you do this on type(request) then you can put class identifiers in the break table